### PR TITLE
Alert and button complete

### PR DESCRIPTION
**Pull request by MiheerTamkhane**

### CODE REVIEW

**Review by vijaytembugade — APPROVED** [review]
Amazing colors and shades.

**Review by MarkVed17 — APPROVED** [review]
All the components look very clean. Also, great work done there with positioning 🙌🏼

**Review by mimansha-swarup — APPROVED** [review]
looks good to me

File: alert/alert.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import url("/style.css");
+.material-icons {
+  margin: 0 0.5rem;
+  cursor: pointer;
+}
```
Comment (horsemaker) [inline alert/alert.css:6]: I am not sure, but I guess it's not a good practice to play with the original (library) class. You could rather add a new class to make the necessary changes. Check it with others also.

**Review by horsemaker — APPROVED** [review]
Looks good! 👍

File: alert/alert.css (hunk 1 above)
Comment (MiheerTamkhane) [reply alert/alert.css:6]: Ohh thanks buddy. I didn't knew that

**Pull request merged**